core/src/executor/evaluate/function.rs

```diff
@@ -236,6 +236,28 @@ pub fn abs(name: String, n: Evaluated<'_>) -> Result<Value> {
     }
 }
 
+pub fn ifnull(expr1: Evaluated<'_>, expr2: Evaluated<'_>) -> Result<Value> {
+    // should we pass expr2 as an Expr enum and evaluate it if expr1 is null?
+    // that would be more efficient than evaluating it before hand..
+    // what do you think?
+    match expr1.try_into()? {
+        Value::Bool(b) => Ok(Value::Bool(b)),
+        Value::Date(d) => Ok(Value::Date(d)),
+        Value::I8(n) => Ok(Value::I8(n)),
+        Value::I64(v) => Ok(Value::I64(v.abs())),
+        Value::Decimal(v) => Ok(Value::Decimal(v.abs())),
+        Value::F64(v) => Ok(Value::F64(v.abs())),
+        Value::Interval(i) => Ok(Value::Interval(i)),
+        Value::List(l) => Ok(Value::List(l)),
+        Value::Map(m) => Ok(Value::Map(m)),
+        Value::Str(s) => Ok(Value::Str(s)),
+        Value::Timestamp(t) => Ok(Value::Timestamp(t)),
+        Value::Time(t) => Ok(Value::Time(t)),
+        Value::Uuid(u) => Ok(Value::Uuid(u)),
+        Value::Null => expr2.try_into(),
+    }
```

Comment:
`Evaluated` has `is_null` method.
We can use `is_null` in here.
https://github.com/gluesql/gluesql/blob/c716a72f31f6bc94d2f019c0c4e2e136c2d3bbdb/core/src/executor/evaluate/evaluated.rs#L225-L230

Reply:
ok.